DataFormats/PortableTestObjects/src/alpaka/classes_serial.h

```diff
@@ -2,3 +2,14 @@
 #include "DataFormats/Portable/interface/Product.h"
 #include "DataFormats/PortableTestObjects/interface/TestHostCollection.h"
 #include "DataFormats/PortableTestObjects/interface/TestSoA.h"
+#include "DataFormats/Portable/interface/PortableCollectionCommon.h"
+
+namespace ROOT::Meta::Selection {
+  namespace portablecollection {
+    class c0 : CollectionImpl<0, void> {};
```

Comment:
should this one really be in `PortableTestObjects` ?

Reply:
Like this? 
It also works, we just need to create the inheritance to get the class in the dictionary, there is no effect beyond this file.

```C++
namespace ROOT::Meta::Selection {
  namespace PortableTestObjects  {
    using portablecollection::CollectionLeaf;
    class CL0_TestHostMultiCollection : CollectionLeaf<0, portabletest::TestSoALayout<128, false>> {};
```

Reply:
what I meant is that `CollectionImpl<0, void>` does not seem specific to `DataFormats/PortableTestObjects`, so shouldn't it be in a more general package ?

Reply:
(Correction: `class c0 : CollectionImpl<0, void> {};` is needed, I removed it too fast)